src/messages/MFSMap.h

```diff
@@ -48,11 +50,11 @@ class MFSMap : public Message {
   void decode_payload() {
     bufferlist::iterator p = payload.begin();
     ::decode(epoch, p);
-    ::decode(encoded, p);
+    ::decode(fsmap, p);
   }
   void encode_payload(uint64_t features) {
     ::encode(epoch, payload);
-    ::encode(encoded, payload);
+    ::encode(fsmap, payload, features);
```

Comment:
I wasn't talking about using the feature bits. There's a difference between encoding an FSMap into the payload, and encoding a bufferlist (of an encoded FSMap) into the payload. (There is the usual wrapping around a bufferlist, right? I don't think it's just an append.)